### DISCUSSION TITLE
Inline action button

### DISCUSSION THREAD

**Discussion by kennyhorna** [awcodes/filament-table-repeater#57] [Q&A]
Hi 👋🏼 

Is there a way to inline the action button?
![screenshot_2023-08-12 at 12 41 52@2x](https://github.com/awcodes/filament-table-repeater/assets/16125117/2b7f129f-974b-4579-aa09-e7cdca8ade43)

**Comment by awcodes** [comment]
Wouldn't this look odd as you add items since the button would no longer be inline with the main axis?

**Reply by kennyhorna** [reply]
I think it could be kept in line with the latest row
